pkg/plugin/v2/scaffolds/edit.go

```diff
@@ -56,21 +61,26 @@ func (s *editScaffolder) Scaffold() error {
 			str,
 			"COPY api/ api/",
 			`COPY apis/ apis/`)
-		if err != nil {
+		if err != nil && s.multigroup != config.MultiGroup {
 			return err
 		}
 	} else {
 		str, err = ensureExistAndReplace(
 			str,
 			"COPY apis/ apis/",
 			`COPY api/ api/`)
-		if err != nil {
+		if err != nil && s.multigroup != config.MultiGroup {
 			return err
 		}
 	}
-	// false positive
-	// nolint:gosec
-	return ioutil.WriteFile(filename, []byte(str), 0644)
+
+	if s.multigroup != config.MultiGroup {
+		// false positive
+		// nolint:gosec
+		return ioutil.WriteFile(filename, []byte(str), 0644)
+	}
+
+	return nil
```

Comment:
Following my suggestiion: 


```go
// Scaffold implements Scaffolder
func (s *editScaffolder) Scaffold() error {
	filename := "Dockerfile"
	bs, err := ioutil.ReadFile(filename)
	if err != nil {
		return err
	}
	str := string(bs)

	// update dockerfile
	if s.multigroup {
		str, err = ensureExistAndReplace(
			str,
			"COPY api/ api/",
			`COPY apis/ apis/`)
	} else {
		str, err = ensureExistAndReplace(
			str,
			"COPY apis/ apis/",
			`COPY api/ api/`)
	}
	// ignore the error faced to replace when it 
	// is because the file is already in the desired format
	if err != nil && s.config.MultiGroup != s.multigroup {
		return err
	}
	
	s.config.MultiGroup = s.multigroup
	// false positive
	// nolint:gosec
	return ioutil.WriteFile(filename, []byte(str), 0644)
}
```

Reply:
Done, with add the check `if str != ""` before writing to the file.

Reply:
Added comments